nodejs/README.md

```diff
@@ -2,55 +2,18 @@
 
 ## Installing
 
-The tutorials are using CNCF Jaeger (https://github.com/jaegertracing/jaeger) as the tracing backend, 
+The tutorials are using CNCF Jaeger (https://github.com/jaegertracing/jaeger) as the tracing backend,
 [see here](../README.md) how to install it in a Docker image.
 
-Install the dependencies:
+First clone the [opentracing-tutorial](https://github.com/yurishkuro/opentracing-tutorial/tree/master/nodejs), then install the dependencies:
```

Comment:
since it's not possible to clone a subdirectory, it's better to point to the repo root https://github.com/yurishkuro/opentracing-tutorial

Reply:
Ha yes, that was what I was intending :)